docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,96 @@
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory service used for prototyping and development purpose. You can find the code for the [Tinylicious service](https://github.com/microsoft/FluidFramework/tree/main/server/tinylicious) and [Tinylicious client](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/tinylicious-client). It can be instantiated against both `AzureClient` and `TinyliciousClient` for testing purpose.
 
 ## Using Tinylicious locally
 
+You can run tinylicious locally by executing the following command:
+
+```sh
+npx tinylicious@latest
+```
+
+By default, Tinylicious runs on port `7070`, however you can change port by setting an environment variable as `PORT` by running the below command in `Windows Powershell`,
+
+```sh
+$env:PORT=6502
+npx tinylicious@latest
+```
+
+Now, you can navigate to `http://localhost:6502` on your browser to see the Tinylicious service up and running.
+
 ## How to deploy using Tinylicious
 
+The `AzureClient` supports both instantiating against a deployed Azure Fluid Relay service instance for production scenarios, as well as against a local, in-memory service instance, known as Tinylicious, for development purposes.
+
+`AzureClient` can be connected to a `Azure local service (Tinylicious)` instance by passing in the tenant ID as `local`, the orderer and storage URLs  pointing to the Tinylicious instance on the default values of `http://localhost:7070`. You can use `InsecureTokenProvider` for token resolution while running the service locally for development purposes.
+
+In the below code snippet, `AzureClient` is pointing to Tinylicious service. This is providing you with an added feature to work with both the local Tinylicious service and the deployed Azure Fluid Relay service. `TinyliciousClient` only works with local Tinylicous service.
+
+```typescript
+import { AzureClient, AzureConnectionConfig } from "@fluidframework/azure-client";
+import { InsecureTokenProvider } from "@fluidframework/test-client-utils";
+
+const user = {
+    id: "UserId",
+    name: "Test User",
+};
+
+const config: AzureConnectionConfig = {
+    tenantId: "local",
+    tokenProvider: new InsecureTokenProvider("fooBar", user),
+    // if you're running Tinylicious on a non-default port, you'll need change these URLs
+    orderer: "http://localhost:7070",
+    storage: "http://localhost:7070",
+};
+
+const clientProps = {
+  connection: config,
+}
+
+const client = new AzureClient(clientProps);
+```
+
+To launch the local Tinylicious service instance, run `npx @fluidframework/azure-local-service@latest` from your terminal window.
+
 ## Using custom domains with Tinylicious
+
+You can use `ngrok` which lets you expose port on your local machine to the internet. Ngrok gives you a random hostname for each tunnel. This tool enables you to attach domains/subdomains against those tunnels.
+
```

Comment:
```suggestion
When testing, it can be useful to make Tinylicious available outside localhost. You can use a service like [ngrok](https://ngrok.com/) to expose the Tinylicious port on your local machine to the internet. ngrok gives you a random hostname for each tunnel you create and routes requests to your locally-running Tinylicious service.

```